src/DGtal/topology/CubicalComplex.ih

```diff
@@ -31,6 +31,7 @@
 #include <cstdlib>
 #include <queue>
 #include "DGtal/base/SetFunctions.h"
+#include "DGtal/topology/CubicalComplexFunctions.h"
```

Comment:
I think we can directly include it in `CubicalComplex.h`

Reply:
Ok

Reply:
Never mind, it will be easier (to solve the interdependencies) to include it in the `.ih` or at the end of the `.h`...

Reply:
Too late :)

Reply:
If you want the code to compile, it is never too late :sweat_smile:

Reply:
Done in last commit